### ISSUE TITLE
ColdFusion CFID/CFToken are not persistent in any browser 

### ISSUE THREAD

**Issue by donniebox** [phonegap/phonegap-app-desktop#827]
I am having issues testing in a browser (i.e. Chrome, Firefox ).  I have a login page that submits one set of CFID/CFTOKEN values and the next page that is loaded via location.href is sending a different set of CFID/CFTOKEN values.  This is causing my user specific info AJAX request to fail.  This is only happening in the PhoneGap Desktop.  It is working correctly in the PhoneGap Mobile app.

I need this functionality to work so that I can track the AJAX requests to determine why other features are not working as expected on the mobile version of PhoneGap

**Comment by surajpindoria** [comment]
Hi @donniebox,

I don't know enough about ColdFusion to help you with your specific issues on the browser. But there are ways that you can debug apps running on your mobile device, including inspecting the network traffic. This may be your best bet.

https://cordova.apache.org/docs/en/latest/guide/next/#debugging-cordova-apps

**Comment by donniebox** [comment]
Hi @surajpindoria,

Thanks for the link.  I'll check it out.

**Comment by donniebox** [comment]
I forgot I had this issue opened. 

I figured out the issue.  It was only happening in the browser and it was happening because the cordova proxy wasn't being used for my ajax calls that were running in $( document ).ready() and the ColdFusion session cookies were being reset; hence, the session on the ColdFusion server was being reset.  I moved the ajax calls to a function that is called from the phonegap deviceready event and the ajax calls began using the proxy and the session cookies didn't change.
